lnd.go

```diff
@@ -538,6 +565,21 @@ func Main(cfg *Config, lisCfg ListenerCfg, implCfg *ImplementationCfg,
 	}
 	defer atplManager.Stop()
 
+	// If --tlsencryptkey is set, we previously generated a throwaway
+	// TLSConfig. Now we want to remove that and load the persistent
+	// TLSConfig. The wallet is unlocked at this point so we can use the
+	// real KeyRing.
+	if cfg.TLSEncryptKey {
+		err := tlsManager.LoadPermanentCertificate(
+			activeChainControl.KeyRing,
+		)
+		if err != nil {
+			err := fmt.Errorf("unable to reload TLS credentials: "+
+				"%v", err)
+			ltndLog.Error(err)
```

Comment:
Since we're just logging the error, can be `ltndLog.Errorf("unable to reload ... %v", err)` directly.